homebrew_challenge/src/App.jsx

```diff
@@ -365,113 +365,198 @@ const App = () => {
-              <Col>
-                {sortedTeamMembers.slice(0, sortedTeamMembers.length/2+1)}
-              </Col>
-              <Col>
-                {sortedTeamMembers.slice(sortedTeamMembers.length/2+1, sortedTeamMembers.length)}
-              </Col>
-            </Row>
-          </Container>
-        <hr/>
+        <p className="px-5">
+          The Covid-19 dashboard for Scotland was created by the following STA
+          volunteers:{" "}
+        </p>
+        <Container className="team-members">
+          <Row>
+            <Col>
+              {sortedTeamMembers.slice(0, sortedTeamMembers.length / 2 + 1)}
+            </Col>
+            <Col>
+              {sortedTeamMembers.slice(
+                sortedTeamMembers.length / 2 + 1,
+                sortedTeamMembers.length
+              )}
+            </Col>
+          </Row>
+        </Container>
+        <hr />
+      </div>
+    );
+  }
+
+  function pageAccessibility() {
+    return (
+      <div fluid="true" className="accessibility-page">
+        <h1>Accessibility</h1>
+        <p className="px-5">
+          We want everyone who visits the Covid-19 Dashboard to feel welcome,
+          and able to find information whatever their circumstances
+        </p>
+        <p className="px-5">To do this we've:</p>
+        <ul>
+          <li>
+            Designed the dashboard to be accessible and usable for everyone
+          </li>
+          <li>
+            Provided alternative text and audio for media content where
+            appropriate
+          </li>
+        </ul>
+        <hr />
+        <h2>Confirmance statement</h2>
+        <p className="px-5">
+          We're working towards meeting the AA standard of the Web COntent
+          Accessibility Guidelines (WCAG) 2.0.
+        </p>
+        <p className="px-5">
+          Automated testing has taken place against these standards and all
+          expectations are met under the directive
+        </p>
+        <hr />
+        <h2>Browsers</h2>
+        <p className="px-5">
+          We've tested the website using the following web browsers:
+        </p>
+        <ul className="browsers">
+          <li>Google Chrome</li>
+          <li>Mozilla Firefox</li>
+          <li>Safari</li>
+          <li>Microsoft Edge</li>
+        </ul>
+        <p className="px-5">
+          We've tested the website using the following mobile browsers:
+        </p>
+        <ul className="browsers">
+          <li>Google Chrome</li>
+          <li>Mozilla Firefox</li>
+          <li>Safari</li>
+          <li>Samsung Browser</li>
+        </ul>
+        <hr />
+        <h2>Screen Readers</h2>
+        <p className="px-5">
+          This website is compatible with modern screen readers and other
+          assistive technologies
+        </p>
+        <hr />
+        <h2>Data Sonification</h2>
+        <p className="px-5">
+          <i>
+            (Some copy needed from Eirini around the sonification piece to go
```

Comment:
Can we get this piece sorted? Either get the text or remove the place holder for now.